.github/workflows/dart_ci.yml

```diff
@@ -50,17 +50,18 @@ jobs:
       - name: Print Dart SDK version
         run: dart --version
 
-      - name: Delete Dart-2-only files when running on Dart 3
+      - id: install
+        name: Install dependencies
+        run: dart pub get
```

Comment:
Moved `pub get` one step earlier in all these, since `update_tests_for_dart_3.sh` reads the resolved version in pubspec.lock